rust/gui-client/src-tauri/src/client/gui/system_tray.rs

```diff
@@ -510,48 +484,38 @@ mod tests {
                     .copyable("test")
                     .copyable(GATEWAY_CONNECTED),
             )
+            .add_submenu(
+                "Internet Resource",
+                Menu::default()
+                    .separator()
+                    .disabled(INTERNET_RESOURCE_DESCRIPTION)
+                    .separator()
+                    .disabled("Site")
+                    .copyable("test")
```

Comment:
Is this intentional or leftover from testing?

Reply:
These are the tests, the screenshots doesn't show the sites because I haven't set the site for my mock resource.

Do you think the internet resource shouldn't show the site status?

Reply:
Ah ok. Nevermind the above then.